Jellyfin.Server/Jellyfin.Server.csproj

```diff
@@ -13,9 +13,6 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <Nullable>enable</Nullable>
-
-    <!-- Used for generating migrations for EF Core -->
-    <GenerateRuntimeConfigurationFiles>True</GenerateRuntimeConfigurationFiles>
```

Comment:
why is it removed?

Reply:
Apparently it's not needed, I added it because I saw something about that online and things started working, but then I removed it and everything still works, so 🤷. Also the Jellyfin.Server project isn't even where it would be added if it was necessary, so it's a mystery why adding that ever fixed anything